src/natcap/invest/ui_server.py

```diff
@@ -249,3 +250,8 @@ def log_model_exit():
         payload['session_id'],
         payload['status'])
     return 'OK'
+
+@app.route(f'/{PREFIX}/languages', methods=['GET'])
+def get_supported_languages():
+    """Return a mapping of supported languages to their display names."""
+    return json.dumps(natcap.invest.LOCALE_NAME_MAP)
```

Comment:
Hmm, are you sure you have the latest version of `natcap.invest` from this branch installed? `LOCALE_NAME_MAP` is directly derived from the contents of `src/natcap/invest/internationalization/locales`, which no longer contains Chinese on this branch.

Reply:
Okay, I was a little confused, `40d216b` _was_ the latest commit at the time. So I was up-to-date, but I guess I had competing installations or something because I had to trash my `invest/build` and `dist` folders to get the correct `LOCALE_NAME_MAP`.